app/models/team.rb

```diff
@@ -88,6 +88,17 @@ def add_member(user, _assignment_id = nil)
       TeamUserNode.create(parent_id: parent.id, node_object_id: t_user.id)
       add_participant(parent_id, user)
       ExpertizaLogger.info LoggerMessage.new('Model:Team', user.name, "Added member to the team #{id}")
+      # if assignment_id is nil, then don't send an assignment name
+      assignment_name = assignment_id ? Assignment.find(assignment_id).name.to_s : ''
+      # Now that a new team member has been added to a team, send an email to them letting them know
+      if MentorManagement.user_a_mentor?(user)
+        MailerHelper.send_team_confirmation_mail_to_user(user, '[Expertiza] Added to a Team', 'mentor_added_to_team', name.to_s, assignment_name).deliver
+      elsif !user.is_a?(Participant)
```

Comment:
But a mentor is also a participant in the assignment.  I don't understand this line.